### ISSUE TITLE
snapToGrid fix?

### ISSUE THREAD

**Issue by guillermofr** [dotnetCarpenter/mua#1]
Are you going to fix this? thx for your work

**Comment by guillermofr** [comment]
solved 
const line = drawing.polyline(null, {snapToGrid:0}).draw({snapToGrid:20});

**Comment by dotnetCarpenter** [comment]
yep. This project is _work in progress_. I'm doing it for friend, to test the svg.draw.js plugin and for fun. I didn't expect anyone to use it. But great if it's useful to you!

**Comment by dotnetCarpenter** [comment]
@guillermofr BTW, if you have any other wishes, I will incorporate them eventually.

As I said it's a more of a demo project to iron out the svg.draw.js API with @fuzzyma. I don't have that much time to spend on it.
